tests/test_build_gettext.py

```diff
@@ -76,7 +76,7 @@ def test_all(app, status, warning):
     yield assert_equal, _("Testing various markup"), u"Testing various markup"
 
 
-@with_app('gettext', testroot='intl',
+@with_app('gettext', testroot='intl', srcdir='gettext',
```

Comment:
This should be replaced by `pytest.mark.sphinx` because you marked `with_app` decorator as deprecated.